src/Illuminate/Auth/Access/Gate.php

```diff
@@ -123,19 +128,27 @@ public function define($ability, $callback)
      * @param  string  $name
      * @param  string  $class
      * @param  array   $abilities
+     * @param  array   $options
      * @return $this
      */
-    public function resource($name, $class, array $abilities = null)
+    public function resource($name, $class, array $abilities = [], array $options = [])
     {
         $abilities = $abilities ?: [
+            'index'   => 'index',
             'view'   => 'view',
             'create' => 'create',
             'update' => 'update',
             'delete' => 'delete',
         ];
 
+        $options += ['guestable' => ['index', 'view']];
```

Comment:
most of the time you need `index` & `view` to be enabled for guests thats why I though it's better to be default